easysocial/easysocial/votes.php

```diff
@@ -65,16 +65,26 @@ public function getPollData()
 	/**	  
 	 * Function userful for vote
 	 * 	 
-	 * @return  JSON	 
+	 * @return  object 
 	 */
-	public function processVote()
+	private function processVote()
 	{
-		$app = JFactory::getApplication();
-		$pollId = $app->input->get('id', 0, 'int');
-		$itemId  = $app->input->get('itemId', 0, 'int');
-		$action  = $app->input->get('act', '', 'default');
+		$app		=	JFactory::getApplication();
+		$pollId		=	$app->input->get('id', 0, 'int');
+		$itemId		=	$app->input->get('itemId', 0, 'int');
+		$action		=	$app->input->get('act', '', 'default');
```

Comment:
@Pratikshakhandagale  - There noting like `default` filter. Use on of these - https://docs.joomla.org/Retrieving_request_data_using_JInput